### DISCUSSION TITLE
max-width 1920px

### DISCUSSION THREAD

**Discussion by marcstorch** [wearebraid/griddle#5] [Q&A] [answered]
hi,
thanks for releasing griddle! i really like it so far :)
can you give me the best practice for expanding the maxwidth?
iam running into some strange issues increasing the $g-max-column-width to 5.5em and above.
on resolutions wider than 2000px the grid container expands to the full viewport width.

thanks!

**Comment by andrew-boyd** [comment, marked as answer]
Hi @marcstorch, would you be able to create a reproduction in something like code sandbox? One thing you could try (suspected issue) is to create a new breakpoint that is larger than the target max body width. 

maybe add something like `'xxl' 200em 3em` to your breakpoints array? I wonder if the `max-body` automatic breakpoint is not being inserted correctly for you given the larger target size you're going for.

**Reply by marcstorch** [reply]
hi @andrew-boyd.
thanks a lot! works like a charm.

**Reply by andrew-boyd** [reply]
Glad to hear it! I'll look into my `max-body` insertion logic and see if I can make sure it gets put on the end of the list even if it's the largest breakpoint.
